### DISCUSSION TITLE
Adding new bib entry but does not show up on website.

### DISCUSSION THREAD

**Discussion by jS5t3r** [alshedivat/al-folio#2538] [Q&A]
Hello, 

I need help. 
I added a new bib entry in `_bibliography/papers.bib`.
All checks are successful. 
The nw bib entry does not show up on the website. What I may change to improve this?

**Comment by george-gca** [comment]
Does your bib entry have `year`?

**Reply by jS5t3r** [reply]
Hi, I  think that the problem is more I am not used to the github workflows. 
After some time it just showed up. Maybe after using prettier etc.
